src/IO/SigGenInterface.jl

```diff
@@ -35,7 +35,7 @@ function readsiggen(file_path::String; dicttype::Type = Dict{String,Any}, T::Typ
         "inner_taper_width"    => 0,
         "taper_angle"          => 0,
         "taper_length"         => 0,
-        "Li_thickness"         => 0,
+        "lithium_thickness"         => 0,
```

Comment:
I don't think anything should be changed in this file at all. This might break stuff...